models/boil_queries_test.go

```diff
@@ -23,7 +23,7 @@ func MustTx(transactor boil.Transactor, err error) boil.Transactor {
 	return transactor
 }
 
-var rgxPGFkey = regexp.MustCompile(`(?m)^ALTER TABLE ONLY .*\n\s+ADD CONSTRAINT .*? FOREIGN KEY .*?;\n`)
+var rgxPGFkey = regexp.MustCompile(`(?m)^ALTER TABLE ONLY .*\n\s+ADD CONSTRAINT .*? FOREIGN KEY .*?;`)
```

Comment:
This whole file is automatically generated, so this edit is kind of problematic.
Next time we will import boiler files from MDB (which we do seldom) this will break again.